<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajout</class>
 <widget class="QDialog" name="Ajout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dessinateur</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QTableWidget" name="w_dessinateur">
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Nouvelle colonne</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTableWidget" name="w_auteur">
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Nouvelle ligne</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Nouvelle colonne</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Auteur</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Titre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Ajout</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Ajout</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
